mllib/src/main/scala/org/apache/spark/mllib/feature/ChiSqSelector.scala

```diff
@@ -263,9 +278,15 @@ object ChiSqSelector {
   /** String name for `fpr` selector type. */
   private[spark] val FPR: String = "fpr"
 
+  /** String name for `fdr` selector type. */
+  private[spark] val FDR: String = "fdr"
```

Comment:
I know this applies to the existing line above too, but, this comment isn't descriptive. You can spell out what all of these mean if there's javadoc at all here.